ansible-cli.js

```diff
@@ -0,0 +1,195 @@
+const { promisify } = require("util");
+const childProcess = require("child_process");
+const _ = require("lodash");
+const {
+  prependNamedArguments,
+  createDockerVolumeConfig,
+  extractMountPointsFromVolumeConfigs,
+  mergeVolumeConfigsEnvironmentVariables,
+  validatePaths,
+  createDockerVolumesString,
+  createEnvironmentVariablesString,
+} = require("./helpers");
+const { ANSIBLE_DOCKER_IMAGE } = require("./consts.json");
+
+const exec = promisify(childProcess.exec);
+
+async function execute({
+  config: unparsedAnsibleConfig,
+  command,
+}) {
+  const {
+    volumeConfigs,
+    environmentVariables,
+    ansibleConfig,
+  } = await parseAnsibleConfig(unparsedAnsibleConfig);
```

Comment:
A function should return only one result, but let's keep that for the second round of review.

Considering that the `unparsedAnsibleConfig` are actually some kind of params, it makes much more sense to produce config from it, what's pretty much reflected in this function name.